pkg/operator/controllers/workaround/routefix.go

```diff
@@ -39,96 +36,61 @@ do
 done`
 )
 
-type RouteFix interface {
-	CreateOrUpdate(ctx context.Context) error
-}
+var _ Workaround = (*routeFix)(nil)
 
 type routeFix struct {
-	log *logrus.Entry
-	env env.Interface
-
-	cli    kubernetes.Interface
-	seccli securityclient.Interface
+	log          *logrus.Entry
+	cli          kubernetes.Interface
+	seccli       securityclient.Interface
+	versionFixed *version.Version
 }
 
-func New(log *logrus.Entry, e env.Interface, cli kubernetes.Interface, seccli securityclient.Interface) RouteFix {
+func NewRouteFix(log *logrus.Entry, cli kubernetes.Interface, seccli securityclient.Interface) Workaround {
+	versionFixed, _ := version.ParseVersion("4.99.0") //TODO: remove when version is known
```

Comment:
Please raise a follow-up issue in GH otherwise we gonna lose it :) with reference to this PR

Reply:
also, no need to put in an arbitrary version here right now or have the versionFixed field - just return true in IsRequired for now.